app/services/search_service.rb

```diff
@@ -182,4 +182,29 @@ def textSearch_questions(srchString)
     end
     sresult
   end
+
+  def nearbyNodes(srchString)
+    sresult = DocList.new
+    coordinates = srchString.split(",")
+    lat = coordinates[0]
+    lon = coordinates[1]
+
```

Comment:
latitude, longitude = coordinates 
directly we can assign, ruby will take care of it.

Reply:
You are absolutely right but I think for sake of clarity of all writing one extra line is good

Reply:
Good suggestion! I always forget to do this myself.